CHANGELOG.md

```diff
@@ -4,7 +4,7 @@
 
 fix(errors): `IsOTPRequired` detects 'OTP Required' API errors
 
-## 6.7.5
```

Comment:
Nope. This shouldn't be changed. A new title must be added above for the new version

Reply:
Mb, fixed